### PR TITLE
TA-11416. Behavior.Affix patch refresh on resize.

### PR DESCRIPTION
**Pull request by theElectricMiracleCat**
@anutron || @davywentworth

### CODE REVIEW

**Comment by anutron** [comment]
I think you should assign this to @davywentworth as I wrote a lot of it.

**Comment by anutron** [comment]
It looks good to me, but good to get other eyes on it.

File: js/Source/Bootstrap.Affix/Bootstrap.Affix.js [hunk 1]
```diff
@@ -187,5 +207,5 @@ Bootstrap.Affix.update = function(instance, y, monitorSize){
   if (y < top && instance.pinned) instance.unpin();
   // if we've scrolled past the bottom line, unpin
   else if (bottom && bottom < y && y > top && instance.pinned) instance.unpin(true);
-  else if (y > top && (!bottom || (bottom && y < bottom)) && !instance.pinned) instance.pin();
-};
\ No newline at end of file
+  else if (y >= top && (!bottom || (bottom && y < bottom)) && !instance.pinned) instance.pin();
```
Comment (davywentworth) [inline js/Source/Bootstrap.Affix/Bootstrap.Affix.js]: was there a hitch or something right at y?

**Comment by davywentworth** [comment]
LGTM

**Pull request closed without merging**